cmp/7e7d33d5-7101-47f6-a358-1126c7876aa4/component.lp

```diff
@@ -0,0 +1,94 @@
+(librepcb_component 7e7d33d5-7101-47f6-a358-1126c7876aa4
+ (name "Buffer With Inverted Enable Quad")
+ (description "Quad buffer/line driver; 3-state")
+ (keywords "74HCT125, 74HC125, 74XX125, buffer, driver, line-driver, quad-buffer, quad buffer")
+ (author "Shibin Das")
+ (version "0.1")
+ (created 2019-04-07T20:07:35Z)
+ (deprecated false)
+ (category 56f01b1d-0723-4c33-88f7-9968cf96b50f)
+ (category a4c3b94b-5321-4adb-8a77-d3b53f178cee)
+ (schematic_only false)
+ (default_value "74XX125")
+ (prefix "IC")
+ (signal a10f32b8-dec9-4894-92ff-ba1ae5e239c0 (name "2") (role passive)
```

Comment:
Please rename all component signals to their actual functionality, e.g. "IN1", "OUT1", "/OE1" for the first gate. See the OpAmp Quad as an example. This is needed to make components much more generic. The pinout is a matter of the device, not of the component.